src/metadata.jl

```diff
@@ -52,3 +52,20 @@ function _image()
 
     return _CACHE[:IMAGE]
 end
+
+function Pkg.activate(jlso::JLSOFile, path; kwargs...)
+    ispath(path) || mkpath(path)
+    open(io -> Pkg.TOML.print(io, jlso.project), joinpath(path, "Project.toml"), "w")
+    open(io -> Pkg.TOML.print(io, jlso.manifest), joinpath(path, "Manifest.toml"), "w")
+    Pkg.activate(path; kwargs...)
+end
+
+function Pkg.activate(f::Function, jlso::JLSOFile, path; kwargs...)
```

Comment:
should make `path` optional and defaulting to `mktmp()`
here and above

Reply:
👌